package.json

```diff
@@ -23,13 +29,17 @@
   "devDependencies": {
     "@babel/eslint-parser": "^7.17.0",
     "@babel/preset-env": "^7.16.11",
-    "@playwright/test": "^1.20.0",
+    "@playwright/test": "^1.40.0",
     "cross-env": "^7.0.3",
+    "env-cmd": "^10.1.0",
```

Comment:
Why do you need both env-cmd and dotenv? Also, doesn't playwright do this automatically?

Reply:
removed the envcmd